src/view/actions/sendEvent.jsx

```diff
@@ -10,48 +10,43 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import "regenerator-runtime"; // needed for some of react-spectrum
 import React from "react";
 import { object, string } from "yup";
-import Textfield from "@react/react-spectrum/Textfield";
-import Checkbox from "@react/react-spectrum/Checkbox";
-import ComboBox from "@react/react-spectrum/ComboBox";
-import Select from "@react/react-spectrum/Select";
-import FieldLabel from "@react/react-spectrum/FieldLabel";
-import "@react/react-spectrum/Form"; // needed for spectrum form styles
-import render from "../render";
-import WrappedField from "../components/wrappedField";
-import ExtensionView from "../components/extensionView";
+import { Form, Item } from "@adobe/react-spectrum";
+import {
+  ComboBox,
+  Picker,
+  TextField,
+  Checkbox
+} from "../components/hookFormReactSpectrum";
+import DataElementSelector from "../components/dataElementSelector";
+import render from "../spectrum3Render";
+import ExtensionView from "../components/hookFormExtensionView";
 import getInstanceOptions from "../utils/getInstanceOptions";
 import singleDataElementRegex from "../constants/singleDataElementRegex";
-import "./sendEvent.styl";
-import InfoTipLayout from "../components/infoTipLayout";
+// import "./sendEvent.styl";
 import DecisionScopesComponent from "../components/decisionScopesComponent";
-
-const decisionScopesOptions = {
-  CONSTANT: "constant",
-  DATA_ELEMENT: "dataElement"
-};
```

Comment:
I broke these out into an enum file so that DecisionScopesComponent could access them directly instead of SendEvent sending them down through props.